compiler/pragmas.nim

```diff
@@ -897,6 +897,12 @@ proc singlePragma(c: PContext, sym: PSym, n: PNode, i: var int,
         incl(sym.flags, sfWasForwarded)
       of wDynlib:
         processDynLib(c, it, sym)
+      of wInternalProc:
+        noVal(c, it)
+        if sym.name == getIdent(c.cache, "nimInternalNewException"):
```

Comment:
you can do cases sensitive string comparison here as well.

Reply:
done, thanks for pointing this out; all i had to do was use `id` field